pkgs/development/php-packages/decimal/default.nix

```
@@ -0,0 +1,28 @@
+{
+  buildPecl,
+  lib,
+  mpdecimal,
+  php,
+}:
+let
+  version = "1.5.0";
+in
+buildPecl {
+  pname = "decimal";
+
+  version = version;
+  sha256 = "it8w8hOLYwtCZoDYhaP5k5TD/pQLtj37K2lSESF80ok=";
```

In new packages, we prefer

```suggestion
  hash = "sha256-it8w8hOLYwtCZoDYhaP5k5TD/pQLtj37K2lSESF80ok=";
```